DfE.FindInformationAcademiesTrusts/Pages/Search.cshtml.cs

```diff
@@ -5,24 +5,31 @@
 
 namespace DfE.FindInformationAcademiesTrusts.Pages;
 
-public class SearchModel : PageModel, ISearchFormModel
+[ValidateAntiForgeryToken]
+public class SearchModel : PageModel, ISearchFormModel, IPaginationModel
 {
     public record AutocompleteEntry(string Address, string Name, string? TrustId);
 
     private readonly ITrustProvider _trustProvider;
     private readonly ITrustSearch _trustSearch;
+    public string PageName { get; } = "Search";
+    public IPagination PageStatus { get; set; }
```

Comment:
Everywhere the `IPagination` interface is used, it is named `PageStatus`. We could rename the interface `IPageStatus` (and the associated type to `PageStatus`) to better reflect what we're actually calling it in the rest of the code